runtime/oti/bcverify.h

```diff
@@ -169,10 +169,29 @@ typedef struct J9BCVAlloc {
 	temp1 = POP; \
 	inconsistentStack |= (temp1 == BCV_BASE_TYPE_TOP);
 
-/* Must be 2 singles or a correctly ordered wide pair */
-#define POP_TOS_PAIR( temp1, temp2 )	\
+#define IS_WIDE_TYPE(temp)	\
+	(J9_ARE_ALL_BITS_SET(temp, BCV_BASE_TYPE_LONG) | J9_ARE_ALL_BITS_SET(temp, BCV_BASE_TYPE_DOUBLE))
+
+#define TYPE_PAIR_CHECK(temp1, temp2)	\
+	{ \
+		UDATA isWideType = IS_WIDE_TYPE(temp2); \
+		inconsistentStack |= IS_WIDE_TYPE(temp1); \
+		inconsistentStack |= ((BCV_BASE_TYPE_TOP == temp1) ? (!isWideType) : isWideType); \
```

Comment:
Please wrap `temp1` in parentheses here.

Reply:
Updated.